### ISSUE TITLE
Loki branch - EL7 - Post Exec execution crashes Asbru

### ISSUE THREAD

**Issue by popxunga** [asbru-cm/asbru-cm#537]
Hi Team ...
Just tried the Post Exec option in one of my ssh connections.
I added the execution of a simple "ls -al" command just for testing:
![image](https://user-images.githubusercontent.com/37672088/77902843-bf0b0080-7279-11ea-8d9e-480f36d96fd1.png)

I see that the output of the ls -al command is sent to the terminal from where Asbru was launched, and then after Asbru crashes. I see these messages in the terminal:

Can't call method "get_window" on an undefined value at /usr/share/asbru-cm/lib/PACTerminal.pm line 3325.

[1]+  Segmentation fault      (core dumped) asbru-cm

Thanks.

**Comment by hanspr** [comment]
Fixed, but just to make it clear.

The Idea of Pre and Post Exec commands is to execute "local" applications before opening the session and after the session is closed "not to execute remote commands".

For example:

+ Pre
    - Start a local apache server
+ Post
    - Open your spreadsheet

Is not to send "ll -a" or the like, the session will not be open on the pre commands, and will not exists on the post command

Have improved the documentation.

**Comment by popxunga** [comment]
Hi @hanspr 

Thanks for the explanation, but that was clear to me already.
As I said in #505:
**The Pre Exec function, executes something before the login process begins.
The Post Exec function, executes something after logout.**
So obviously in both case we are talking about local executions and not remote executions.
The "ls -al" wast just something easy I used to test the function.

Meanwhile I tested with the latest loki build and Asbru doesn't crash anymore. 
So problem solved.

Thanks for your support.
